starport/pkg/cosmosgen/templates/root/index.ts.tpl

```diff
@@ -0,0 +1,87 @@
+// THIS FILE IS GENERATED AUTOMATICALLY. DO NOT MODIFY.
+
+import { OfflineSigner, Registry } from "@cosmjs/proto-signing";
```

Comment:
```suggestion
import { OfflineSigner, Registry, GeneratedType } from "@cosmjs/proto-signing";
```

Reply:
actuilly scratch this one...not needed after all i dont think as Registry should be inferred by the imports